src/test/java/com/bynder/sdk/query/upload/SaveMediaQueryTest.java

```diff
@@ -13,20 +13,21 @@
     public static final String EXPECTED_BRAND_ID = "brandId";
     public static final String EXPECTED_NAME = "name";
     public static final String EXPECTED_MEDIA_ID = "mediaId";
-    public static final String EXPECTED_METAPROPERTY = "metaproperty.id1=value1,metaproperty.id2=value2";
+    public static final String EXPECTED_METAPROPERTY_ID = "metapropertyId";
+    public static final String EXPECTED_OPTION_NAME = "optionName";
 
     @Test
     public void initializeSaveMediaQuery() {
         SaveMediaQuery saveMediaQuery = new SaveMediaQuery(EXPECTED_IMPORT_ID);
         saveMediaQuery.setBrandId(EXPECTED_BRAND_ID);
         saveMediaQuery.setName(EXPECTED_NAME);
         saveMediaQuery.setMediaId(EXPECTED_MEDIA_ID);
-        saveMediaQuery.setMetaproperty(EXPECTED_METAPROPERTY);
+        saveMediaQuery.setMetaproperty(EXPECTED_METAPROPERTY_ID, EXPECTED_OPTION_NAME);
```

Comment:
Don't we want to make use of the new `SaveMediaQuery.setMetaproperties()` here?

Reply:
added a test for that getter/setter as well